guides/development/hack_on_wanda.md

```diff
@@ -0,0 +1,55 @@
+# Hack on Wanda
+
+In order to run Wanda, you will need
+
+1. [Elixir](https://elixir-lang.org/), a programming language used to build scalable and maintainable applications.
+2. [Docker](https://docs.docker.com/get-docker/), a platform for developing, shipping, and running applications in containers.
+3. [Docker Compose](https://docs.docker.com/compose/install/), a tool for defining and running multi-container applications.
+
+## Development environment
+
+A `docker-compose` development environment is provided for easy setup and management of dependencies. To start the development environment, simply run the following command:
+
+```
+docker-compose up -d
+```
+
+This command will start a **Postgres** database and a **RabbitMq** instance, which will be used for storage and communication respectively.
+
+## Install dependencies
+
+Before you can start hacking on Wanda, you will need to install its dependencies. You can do this by running the following command:
+
+```
+mix deps.get
+```
+
+This command will install all the required libraries and dependencies for the application.
+
+## Setup Wanda
+
+After installing the dependencies, you will need to perform some initial setup tasks before you can start Wanda. You can do this by running the following command:
+
+```
+mix setup
```

Comment:
Keep in mind that `mix setup` also runs `deps.get`. 

So we might want to either drop the previous `mix deps.get` section or replace `mix setup` with `ecto.setup`.

In any case I would also give a reference to `mix.exs` for further details, in order to avoid documenting every single task, which very likely we'll forget.

Reply:
Hey yeah this is absolutely correct. I removed a section and added a hint about the mix.exs
You are fully right!